src/Translator/assets/src/translator.ts

```diff
@@ -52,6 +53,18 @@ export function getLocale(): LocaleType {
     );
 }
 
+export function setThrowWhenNotFound(throwWhenNotFound: boolean): void {
+    _throwWhenNotFound = throwWhenNotFound;
+}
+
+export function getThrowWhenNotFound(): boolean {
+    return (
+        _throwWhenNotFound ||
+        document.documentElement.getAttribute('data-symfony-ux-translator-throw-when-not-found') === 'true' || // <html data-symfony-ux-translator-throw-when-not-found="true">
+        false
+    );
+}
```

Comment:
The usage of `_throwWhenNotFound` will be internal, I don't think we need a function.